environment

```diff
@@ -43,8 +49,8 @@ FUS_OWNER_TAG=user@email.com
 
 set +a
 
-if [[ -f "${FUS_HOME}/environment.local" ]]; then
-    source "${FUS_HOME}/environment.local"
+if [[ -f "${FUS_HOME}/deployments/${FUS_DEPLOYMENT_STAGE}/environment.local" ]]; then
```

Comment:
i think it would be worth documenting this in readme, perhaps under an environments sections.